flang/lib/Lower/OpenMP.cpp

```diff
@@ -103,7 +103,17 @@ genOMP(Fortran::lower::AbstractConverter &converter,
             genOMP(converter, eval, simpleStandaloneConstruct);
           },
           [&](const Fortran::parser::OpenMPFlushConstruct &flushConstruct) {
-            TODO("");
+            SmallVector<Value, 4> operandRange;
+            if (auto &ompObjectList =
+                    std::get<std::optional<Fortran::parser::OmpObjectList>>(
+                        flushConstruct.t))
+              genObjectList(*ompObjectList, converter, operandRange);
+            if (auto &flushMemoryClause = std::get<
```

Comment:
Do you plan to update the parser to use `OmpClause` instead. If so and you do it upstream, this will fail when Eric will rebase so just keep a look on it.

Reply:
Sure, I'll keep track of this. And definition adjustments of the `FlushOp`.
Thanks!